pkgs/by-name/ki/kiwitalk/package.nix

```diff
@@ -0,0 +1,133 @@
+{ lib
+, fetchFromGitHub
+, copyDesktopItems
+, stdenv
+, stdenvNoCC
+, rustc
+, rustPlatform
+, cargo
+, cargo-tauri
+, openssl
+, libayatana-appindicator
+, webkitgtk
+, pkg-config
+, makeDesktopItem
+, jq
+, moreutils
+, nodePackages
+, cacert
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kiwitalk";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "KiwiTalk";
+    repo = "KiwiTalk";
+    rev = "v${version}";
+    hash = "sha256-Th8q+Zbc102fIk2v7O3OOeSriUV/ydz60QwxzmS7AY8=";
+  };
+
+  postPatch = ''
+    substituteInPlace $cargoDepsCopy/libappindicator-sys-*/src/lib.rs \
+      --replace "libayatana-appindicator3.so.1" "${libayatana-appindicator}/lib/libayatana-appindicator3.so.1"
+  '';
+
+  pnpm-deps = stdenvNoCC.mkDerivation {
+    pname = "${pname}-pnpm-deps";
+    inherit src version;
+
+    nativeBuildInputs = [
+      jq
+      moreutils
+      nodePackages.pnpm
+      cacert
+    ];
+
+    installPhase = ''
+      export HOME=$(mktemp -d)
+      pnpm config set store-dir $out
+      # cat package.json | jq '.pnpm.supportedArchitectures += { "os": ["linux"], "cpu": ["arm64"] }' | sponge package.json
```

Comment:
Is this how you derived the hashes? If so please add a comment indicating how and why to use it.